lib/ansible/modules/files/acl.py

```diff
@@ -138,10 +138,10 @@
     type: list
     sample: [ "user::rwx", "group::rwx", "other::rwx" ]
 '''
-
+import platform
 import os
```

Comment:
Alphabetize imports please.
```suggestion
 import os
 import platform
```